tests/mock_module/resource.py

```diff
@@ -17,6 +28,36 @@
     DraftVersionResource as DraftVersionResourceBase
 from invenio_drafts_resources.resources import \
     DraftVersionResourceConfig as DraftVersionResourceConfigBase
+from invenio_drafts_resources.resources import \
+    RecordResource as RecordResourceBase
+from invenio_drafts_resources.resources import \
+    RecordResourceConfig as RecordResourceConfigBase
+
+
+class RecordLinksSchema(Schema):
+    """Schema for a record's links."""
+
+    self = Link(
+        template=URITemplate("/api/mocks/{pid_value}"),
+        permission="read",
+        params=lambda record: {'pid_value': record.pid.pid_value}
+    )
+    # TODO: Add delete, files, ...
+
+
+class DraftLinksSchema(Schema):
```

Comment:
Comment: Seems to me that we need a small utility to write these guys:

```python
RecordLinksSchema, DraftLinksSchema = create_link_schemas('/api/mocks')
```

Reply:
Let's wait and see... There are tons of permutation of the arguments...